lib/core/network/api_service.dart

```diff
@@ -127,4 +127,26 @@ class APIService {
       return jsonDecode(response.body) as Map<String, dynamic>;
     });
   }
+
+  Future<bool> submitGivts({
+    required Map<String, dynamic> body,
+    required String guid,
+  }) async {
+    final url = Uri.https(apiURL, '/api/v2/users/$guid/givts');
+    return client
+        .post(
+      url,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      encoding: Encoding.getByName('utf-8'),
+      body: jsonEncode(body),
+    )
+        .then((response) {
+      if (response.statusCode >= 400) {
+        throw Exception('something went wrong :(');
+      }
+      return response.statusCode == 200;
```

Comment:
Is there a specific reasons why only a 200 is true? Or should that always return true if its not >400

Reply:
Well , the android app is only using the status code. If the status code is >=300 is retrying to send them 3 times. I am assuming that's because of the internet connection. Also, before it's caching the givts and on 200 is removing them